packages/cli-app/src/exec.js

```diff
@@ -25,6 +25,7 @@ export const exec = command('exec', {
 
   percy: {
     server: true,
+    projectType: 'app',
```

Comment:
You need to do this in start command above as well right ?

Reply:
doesn't look like it but adding it for consistency.